plugins/topology/package.json

```diff
@@ -41,6 +41,7 @@
     "@patternfly/react-tokens": "4.94.6",
     "@patternfly/react-topology": "4.91.27",
     "classnames": "2.x",
+    "js-yaml": "^4.1.0",
```

Comment:
should we add yarn-lock as well as there is a change here?

Reply:
There was no change in yarn-lock upon this addition.